.storybook/data/tokens.json

```diff
@@ -207,7 +207,7 @@
   "eds-theme-color-button-tertiary-border": "#707070",
   "eds-theme-color-button-tertiary-border-hover": "#707070",
   "eds-theme-color-button-tertiary-border-active": "#21272D",
-  "eds-theme-color-button-tertiary-text": "#6B65E2",
+  "eds-theme-color-button-tertiary-text": "#707070",
```

Comment:
There's one token update in here because the tertiary text was set to purple instead of gray. I assume that was a copy/pasta typo.